x-pack/plugin/esql/compute/src/test/java/org/elasticsearch/compute/aggregation/SumDoubleAggregatorFunctionTests.java

```diff
@@ -109,15 +107,14 @@ public void testSummationAccuracy() {
                 driverContext,
                 new SequenceDoubleBlockSourceOperator(driverContext, DoubleStream.of(values)),
                 List.of(simple(nonBreakingBigArrays()).get(driverContext)),
-                new PageConsumerOperator(page -> results.add(page)),
+                new ResultPageSinkOperator(results::add),
```

Comment:
Thanks for doing this. I missed this previously but didn't noticed as we weren't emitting tracked blocks.